base_layer/core/src/consensus/consensus_constants.rs

```diff
@@ -57,12 +52,24 @@ pub struct ConsensusConstants {
     pub(in crate::consensus) emission_decay: f64,
     /// This is the emission curve tail amount
     pub(in crate::consensus) emission_tail: MicroTari,
-    /// This is the initial min difficulty for the difficulty adjustment
-    min_pow_difficulty: (Difficulty, Difficulty),
     /// The offset relative to the expected genesis coinbase value
     genesis_coinbase_value_offset: MicroTari,
     /// This is the maximum age a monero merge mined seed can be reused
     max_randomx_seed_height: u64,
+    /// This keeps track of the block split targets and which algo is accepted
+    /// Ideally this should count up to 100. If this does not you will reduce your target time.
+    algo_split: HashMap<PowAlgorithm, Algo_Time>,
```

Comment:
nit: `algo_split` implies that this struct only contains information about the split, when it contains other info as well
```suggestion
    proof_of_work: HashMap<PowAlgorithm, PowAlgorithmConstants>,
```